odl/solvers/functional/default_functionals.py

```diff
@@ -124,6 +129,248 @@ def convex_conj(self):
+                                      'or 2')
+
+    @property
+    def convex_conj(self):
+        """The convex conjugate functional of the group L1-norm."""
+        conj_exp = conj_exponent(self.pointwise_norm.exponent)
+        return IndicatorGroupL1UnitBall(self.domain, exponent=conj_exp)
+
+    def __repr__(self):
+        """Return ``repr(self)``."""
+        return '{}({!r}, exponent={})'.format(self.__class__.__name__,
+                                              self.domain,
+                                              self.pointwise_norm.exponent)
+
+
+class IndicatorGroupL1UnitBall(Functional):
+
+    """The convex conjugate to the mixed L1--Lp norm on `ProductSpace`.
+
+    See Also
+    --------
+    GroupL1Norm
+    """
+
+    def __init__(self, vfspace, exponent=None):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        vfspace : `ProductSpace`
+            Space of vector fields on which the operator acts.
+            It has to be a product space of identical spaces, i.e. a
+            power space.
+        exponent : non-zero float, optional
+            Exponent of the norm in each point. Values between
+            0 and 1 are currently not supported due to numerical
+            instability.
+            Default: ``vfspace.exponent``, usually 2.
+
+        Examples
+        --------
+        >>> space = odl.rn(2)
+        >>> pspace = odl.ProductSpace(space, 2)
+        >>> op = IndicatorGroupL1UnitBall(pspace)
+        >>> op([[0.1, 0.5], [0.2, 0.3]])
+        0
+        >>> op([[3, 3], [4, 4]])
+        inf
+
+        Can also change exponent of inner  (p) norm:
+
+        >>> op2 = IndicatorGroupL1UnitBall(pspace, exponent=1)
+        """
+        if not isinstance(vfspace, ProductSpace):
+            raise TypeError('`space` must be a `ProductSpace`')
+        if not vfspace.is_power_space:
+            raise TypeError('`space.is_power_space` must be `True`')
+        self.pointwise_norm = PointwiseNorm(vfspace, exponent)
+        super().__init__(space=vfspace, linear=False, grad_lipschitz=np.nan)
+
+    def _call(self, x):
+        """Return ``self(x)``."""
+        x_norm = self.pointwise_norm(x).ufunc.max()
+
+        if x_norm > 1:
+            return np.inf
+        else:
+            return 0
+
+    @property
+    def proximal(self):
+        """Return the proximal factory of the functional.
+
+        See Also
+        --------
+        proximal_l1 : proximal factory for the L1-norm.
+        """
+        if self.pointwise_norm.exponent == 1:
+            return proximal_cconj_l1(space=self.domain)
+        elif self.pointwise_norm.exponent == 2:
+            return proximal_cconj_l1(space=self.domain, isotropic=True)
+        else:
+            raise NotImplementedError('`proximal` only implemented for p = 1 '
+                                      'or 2')
+
+    @property
+    def convex_conj(self):
+        """The convex conjugate is the the group L1-norm."""
```

Comment:
Not sure I like a docstring that says "The X is Y." For me that would be the second line in the docstring.